tools/sv-report

```diff
@@ -557,13 +557,21 @@ try:
         report = jinja2.Template(
             f.read(), trim_blocks=True, lstrip_blocks=True)
 
+    try:
+        build_id = os.environ['GITHUB_ACTION']
+    except:
+        build_id = 'local'
```

Comment:
We could covere it to

`os.environ.get('GITHUB_ACTION', 'local')`

The second parameter is the default value if the env variable is not found, this will make it a little bit simpler